### DISCUSSION TITLE
Unofficial doesn't seem to work for me

### DISCUSSION THREAD

**Discussion by junkoe** [Zero6992/chatGPT-discord-bot#234] [Q&A] [answered]
I get a 403 forbidden error when trying with unofficial. Just wondering if anyone else has had any luck or perhaps I am missing something. I pulled all the latest from github, checked and installed the requirements, I then built the docker image and it runs but when I try to use it in my discord according to logs I see this:

2023-03-08T05:02:34.346203948Z 2023-03-08 05:02:34 ERROR    src.bot -> Error while sending message: Client error '403 Forbidden' for url 'https://chatgpt.duti.tech/api/conversation'
2023-03-08T05:02:34.346249726Z For more information check: https://httpstatuses.com/403

Am I missing something here or is unofficial not working now for others? I also edited the .env file with my api key, which doesn't work on official, and when I try the unofficial method with the auth session token it fails the same way. I am using a free gpt account btw, I suspect that is part of the issue as well.

**Comment by Zero6992** [comment, marked as answer]
Unofficial is dead, using official model

**Reply by bluey656yt** [reply]
Did it get patched today? Because unofficial was working until yesterday

**Reply by Zero6992** [reply]
> Did it get patched today? Because unofficial was working until yesterday

https://github.com/acheong08/ChatGPT/issues/1092

It doesn't seem likely to be fixed

**Comment by junkoe** [comment]
That makes sense now. So the only way to use it is with a paid plan and using an API key then.

Thank you for clarifying.
